### ISSUE TITLE
Why the restarts?

### ISSUE THREAD

**Issue by daniel-lo** [ioBroker/ioBroker.telegram#75]
I was told I can prevent the messages by deleting the start-stop text in the instance .. but why does it happen in the first place?
![img_20181104_185804](https://user-images.githubusercontent.com/4311558/47968311-66293f80-e068-11e8-91bd-921bae96424c.jpg)

**Comment by BuZZy1337** [comment]
Can you provide a Screenshot of your instance settings?

**Comment by daniel-lo** [comment]
Sure!
![tg1](https://user-images.githubusercontent.com/4311558/47968450-8bb74880-e06a-11e8-95eb-22c934c900c2.png)


![tg2](https://user-images.githubusercontent.com/4311558/47968453-8bb74880-e06a-11e8-9793-8cb9861872d2.png)

**Comment by BuZZy1337** [comment]
I think I misunderstood your question.
The adapter is restarting every day because in the past there was a problem that the adapter stopped working after some time..
You can configure/disable the restart schedule in the instance tab of iobroker.

**Comment by daniel-lo** [comment]
yes .. but .. it restarted several times .. if you look at my screenshot

**Comment by BuZZy1337** [comment]
Maybe there was a problem like no network connection or any other issues..
Did you check the iobroker log for any errors?

**Comment by daniel-lo** [comment]
No .. not yet .. but since you clarified the Cron thing .. I deleted the chat log and will observe another night :-)
